floyd/cli/experiment.py

```diff
@@ -45,16 +45,23 @@ def init(project):
 
 @click.command()
 @click.argument('id', required=False, nargs=1)
-def status(id):
+@click.option('-f', '--format', default='default', help='Returns an experiment as json or default format')
+def status(id, format):
     """
     View status of all or specific run.
     It can also list status of all the runs in the project.
     """
     if id:
         experiment = ExperimentClient().get(id)
+        if format == 'json':
+            floyd_logger.info(experiment.to_dict())
+            return
         print_experiments([experiment])
     else:
         experiments = ExperimentClient().get_all()
+        if format == 'json':
+            floyd_logger.info([e.to_dict() for e in experiments])
```

Comment:
string representation of python object is not the same the json string serialization spec. we should be calling `json.dumps()` on the python list object instead.